src/java.desktop/share/native/libawt/awt/medialib/awt_ImagingLib.c

```diff
@@ -2912,6 +2912,11 @@ static int expandPackedBCR(JNIEnv *env, RasterS_t *rasterP, int component,
     jint   *inDataP;
     int loff[MAX_NUMBANDS], roff[MAX_NUMBANDS];
 
+#if defined(LINUX) && defined(s390x)
+    memset(loff, 0, sizeof(loff));
+    memset(roff, 0, sizeof(roff));
+#endif
+
```

Comment:
Of the various ways this could be fixed this is one of my least preferred.
Either 
1) Disable the warning for this file for this toolchain + platform in the makefile, or
2) Zero it out for all platforms. There's no platform-specific logic here.
Unless you think there's a compiler bug and these are all false positive in which case (1) is definitely the way to go.